encoding/rle/rle_amd64.go

```diff
@@ -9,21 +9,29 @@ import (
 var (
 	encodeInt32IndexEqual8Contiguous func(words [][8]int32) int
 	encodeInt32Bitpack               func(dst []byte, src [][8]int32, bitWidth uint) int
+	encodeBytesBitpack               func(dst []byte, src []uint64, bitWidth uint) int
+	decodeBytesBitpack               func(dst, src []byte, count, bitWidth uint)
 )
 
 func init() {
 	switch {
 	case cpu.X86.HasAVX2:
 		encodeInt32IndexEqual8Contiguous = encodeInt32IndexEqual8ContiguousAVX2
 		encodeInt32Bitpack = encodeInt32BitpackAVX2
+		encodeBytesBitpack = encodeBytesBitpackASM
+		decodeBytesBitpack = decodeBytesBitpackASM
```

Comment:
How about installing these if `cpu.X86.HasBMI2` is true? It would be more accurate than testing for AVX2.

Reply:
This is sweet! Completely missed it, thanks! 😄